R/run_fims_age.R

```diff
@@ -0,0 +1,50 @@
+run_fims_age <- function(data = data_mile1) {
+  
+  # Ensure data is a FIMSFrameAge
+  age_frame <- FIMSFrameAge(data)
+  # Set the parameters to be the model-comparison project
+  fims <- Rcpp::Module("fims", PACKAGE = "FIMS")
+
+  # Set up survey 
+ # Survey index values and observation errors
+survey_index <- new(fims$IndexData)
+survey_index$values <- age_frame@
+survey_index$error <- rep(em_input$cv.survey$survey1, times = om_input$nyr)
+# Survey composition
+survey_comp <- new(fims$AgeCompData)
+survey_comp$values <- as.vector(t(em_input$survey.age.obs$survey1))
+survey_comp$sample_size <- rep(em_input$n.survey$survey1, times = om_input$nyr)
+# Likelihood component settings
+survey_index_comp_nll <- new(fims$Lognormal)
```

Comment:
The current implementation of fleet_nll is hard coded to use dnorm(log(y)...) instead of dlnorm. @ChristineStawitz-NOAA, should this be updated in fleet_nll or is it better to use dnorm(log(y)...) to match what is done in the model comparison project?

Reply:
Moved to #334